example/android/app/build.gradle

```diff
@@ -126,10 +127,10 @@ android {
 }
 
 dependencies {
-    compile project(':react-native-video')
-    compile fileTree(dir: "libs", include: ["*.jar"])
     compile "com.android.support:appcompat-v7:23.0.1"
     compile "com.facebook.react:react-native:+"  // From node_modules
+    compile project(':react-native-video')
+//    compile project(':react-native-video-exoplayer') // uncomment to use exoplayer
```

Comment:
Comment out `react-native-video` and uncomment `react-native-video-exoplayer` to test